### PR TITLE
[Bignum] Bignum/test read write named modulu

### PR DESCRIPTION
**Pull request by xkqian**
## Description

Prepare a function to test the utility functions of read(), write() for named modulu.
Prepare a function to test the random data generate by mbedtls_mpi_mod_random().

## PR checklist

Please tick as appropriate and edit the reasons (e.g.: "backport: not needed because this is a new feature")

- [x] **changelog**  not required
- [x] **backport**  not required
- [x] **tests** not required

### CODE REVIEW

**Review by minosgalanakis — CHANGES_REQUESTED** [review]
Looks ok, minor changes requested

File: tests/suites/test_suite_ecp.function [hunk 1]
```diff
@@ -1602,6 +1605,20 @@ void ecp_mod_read_write(char *input_A, int id, int ctype)
     TEST_EQUAL(limbs, rX.limbs);
     ASSERT_COMPARE(rA.p, bytes, rX.p, bytes);
 
+    memset(bufx, 0x00, bytes);
+    memset(rX_raw, 0x00, bytes);
+    /* Then test big endian write and read */
+    TEST_EQUAL(0, mbedtls_mpi_mod_write(&rA, &m, (unsigned char *) bufx,
+                                        bytes,
+                                        MBEDTLS_MPI_MOD_EXT_REP_BE));
```
Comment (minosgalanakis) [inline tests/suites/test_suite_ecp.function:1613]: Nitpicking, so I wont block the review. Now with bytes, they can both fit in one line:

~~~~~~
    TEST_EQUAL(0, mbedtls_mpi_mod_write(&rA, &m, (unsigned char *) bufx,
                                        bytes, MBEDTLS_MPI_MOD_EXT_REP_BE));
~~~~~~

**Review by minosgalanakis — DISMISSED** [review]
Looks good

**Review by gabor-mezei-arm — APPROVED** [review]
Looks good to me

**Review by gabor-mezei-arm — APPROVED** [review]
LGTM

**Review by minosgalanakis — APPROVED** [review]

**Pull request merged**